### ISSUE TITLE
Testing with custom images of garment only

### ISSUE THREAD

**Issue by dain5832** [cuiaiyu/dressing-in-order#50]
First of all, thanks for the amazing project!

My teammates and I are using your work in part of our course project, and we are using the FashionIQ dataset, which is mostly consisted of garment-only images, for inference on your pretrained model
Can you give us a more detailed explanation of how to preprocess the dataset and run the inference in the case of garment-only images?

We found you mentioned that P_{g_k} is going to be an empty heatmap when the image is garment only,
but I'm not really sure how can we implement it.
And it seems like the parser does not work well on garment-only images, so if there are any other methods we can try to parse the garment from the background.

Also, we are struggling with the face overfitting issue and would like to ask if using garment-only images will help resolve the issue since the pose is not transferred.

**Comment by snaiws** [comment]
face overfitting issue : #32 

garment-only image issue : I'm wondering as well

**Comment by snaiws** [comment]
hey i solved garment-only image issue

treat garment as human model 
but set all keypoints as -1 
and instead of human parsing, use image segmentation. there would be a garment area(lip label) and background(0)
here's example
![jacket3](https://user-images.githubusercontent.com/54572026/188587860-7d91d2ac-0fc7-40bd-8da0-ab08929b67fc.png)

**Comment by dain5832** [comment]
Thanks!! I'll check that out

**Comment by netsalesbg-ltd** [comment]
@snaiws Do you have to retrain the whole network again for garment only? Does retraining fix face overfitting?
